ilastik/applets/dataSelection/opDataSelection.py

```diff
@@ -584,10 +584,13 @@ def from_h5_group(cls, group: h5py.Group, params: Dict = None):
         """
         Returns a deserialized DatasetInfo dict equivalent to a dict from a MultiscaleUrlDatasetInfo
         """
+        from lazyflow.utility.io_util.RESTfulPrecomputedChunkedVolume import RESTfulPrecomputedChunkedVolume
+
+        remote_source = RESTfulPrecomputedChunkedVolume(group["filePath"][()].decode().lstrip("precomputed://"))
```

Comment:
I cannot think of another way currently, but it's a bit odd to access the data before deserializing the metadata fully...

Reply:
Well we can just do the upgrading after default deserialization:

```
        deserialized = super().from_h5_group(group)
        remote_source = RESTfulPrecomputedChunkedVolume(deserialized.url.lstrip("precomputed://"))
        deserialized.nickname = cls._nickname_from_url(deserialized.nickname)
        deserialized.working_scale = remote_source.highest_resolution_key
        deserialized.scale_locked = True
        return deserialized
```

I like it more actually, will do it this way 👍 

But due to this I found a bug that needs to be fixed :D The scale combobox doesn't init at the correct selected option now, and I'm not sure why. But it's bad because it actually resets `working_scale` to the lowest resolution.